src/elements/radio.svelte

```diff
@@ -52,16 +52,30 @@ const handleClick = (value: string) => {
     {/if}
   </div>
 
-  {#each parsedOptions as option}
-    <button
-      bind:this={button}
-      class={cx('capitalize border-y border-l last:border-r border-black px-2 py-1 text-xs', {
-        'bg-white': option !== selected,
-        'bg-black text-white': option === selected,
-      })}
-      on:click={() => handleClick(option)}
-    >
-      {option}
+  <div class="flex">
+    {#each parsedOptions as option}
+      <button
+        bind:this={button}
+        class={cx('capitalize border-y border-l last:border-r border-black px-3 py-2 text-xs', {
```

Comment:
Why did we increase padding here? we're generally trying to keep the UI for app tight / smaller

Reply:
Good point! How's this?

<img width="218" alt="image" src="https://user-images.githubusercontent.com/17263/195901476-2b84cd9b-8a1c-45da-8d4a-14a6b0053be7.png">